PACKAGES_UPDATE.md

```diff
@@ -7,9 +7,9 @@ To udpate our packages use a package called ![npm-check-updates](https://www.npm
 In a terminal, use the followings commands
 
 - To list the available updates, type `npx --yes npm-check-updates`
```

Comment:
it's a little bit extra, but could you add the 2 scripts 
```
        "list-outdated-deps": "pnpm outdated -r --format list",
        "update-outdated-deps": "pnpm update -r --latest \"!eslint\"",
```

We have them in hopper to update dependencies